judge/runguard_test/runguard_test.sh

```diff
@@ -0,0 +1,289 @@
+#!/bin/bash
```

Comment:
I assume you're aware that the other tests use `bats` but decided that this was better?

Reply:
Yeah, my test is allows for more flexibility that I think was necessary here.